mkdocs.yml

```diff
@@ -55,16 +70,31 @@ markdown_extensions:
   - pymdownx.superfences:
       preserve_tabs: true
   - pymdownx.snippets
+  - pymdownx.details
 
 
 nav:
   - Home: index.md
   - Download and installation: installation.md
-  - Custom math example gallery: _autogenerated/custom_math_examples.md
-  - Mathematical formulation:
-    - Base model: _generated/math.md
-    - Storage constraints when clustering time: _generated/math_storage_inter_cluster.md
+  - Inbuilt math:
+    - index: math/index.md
+  - Custom math:
+    - custom_math/index.md
+    - custom_math/syntax.md
```

Comment:
I would change the order, components first, syntax second (see also the suggested addition to `math/index.md`)

Reply:
I think either works. Components first seems more practical since you get to the "real-world application" more quickly, syntax first seems more logical. Happy to leave it as is and update the text in `math/index.md` accordingly.